### PR TITLE
fix: footer position

### PR DESCRIPTION
**Pull request by GusBaamonde**
related to: https://github.com/FromDoppler/Doppler-UI-System/pull/155

footer--position

Let me know what you think: @mvillaseco @santiagopaolucci @andresmoschini

### CODE REVIEW

**Comment by andresmoschini** [comment]
Estuvimos charlando con @GusBaamonde, porque en resoluciones un poco más pequeñas de lo normal, se veía un scroll, incluso, en mi máquina, con el browser maximisado también. 

Él va a revisar alternativas, por ejemplo usando `calc` o un alto mínimo para intentar evitar ese problema.

**Comment by santiagopaolucci** [comment]
I saw this scene with Gus. I agree.

File: public/index.html [hunk 1]
```diff
@@ -30,7 +30,7 @@
     <noscript>
       You need to enable JavaScript to run this app.
     </noscript>
-    <div id="root">
+    <div id="root" class="dp-container">
```
Comment (fcoronelmakingsense) [inline public/index.html]: it's not more clear `app-container` ? or the same

Comment (fcoronelmakingsense) [reply public/index.html]: or `webapp-container`

Comment (GusBaamonde) [reply public/index.html]: The nomenclature of classes that we have been using is `dp` -` component name`, `dp` comes from` doppler`

Comment (andresmoschini) [reply public/index.html]: I understand your concern @fcoronelmakingsense.

I think that you are right in that it is not something _related to our brand_. I saw this class as a utility one, something not related to our brand that could be used by a library client to make things easy.

In that case, what do you think about using a name related to the functionality or behavior related to this element?

The question is: what makes this _container_ different than another container, for example, a simple _div_?

Comment (fcoronelmakingsense) [reply public/index.html]: i talk with @GusBaamonde and we think can use something like `dp-app-container` to mantein the prefix

**Pull request merged**